lib/net.c

```diff
@@ -169,6 +190,86 @@ get_if_back(struct net_config *net_conf)
 	return &net_conf->back;
 }
 
+int
+gatekeeper_setup_rss(uint8_t portid, uint16_t queues[], uint16_t num_queues)
+{
+	int ret = 0;
+	uint32_t i;
+	struct rte_eth_dev_info dev_info;
+	struct rte_eth_rss_reta_entry64 reta_conf[GATEKEEPER_RETA_MAX_SIZE];
+
+	/* Get RSS redirection table (RETA) information. */
+	memset(&dev_info, 0, sizeof(dev_info));
+	rte_eth_dev_info_get(portid, &dev_info);
+	if (dev_info.reta_size == 0) {
+		RTE_LOG(ERR, PORT,
+			"failed to setup RSS at port %hhu"
```

Comment:
Don't break error strings even if they go above the line limit. The reason for this is to enable "grepping" for error messages.